Frontend/ui-driver/src/Components/RideActionModal/View.purs

```diff
@@ -365,14 +366,17 @@ sourceAndDestinationView push config =
 
 startRide :: forall w . (Action -> Effect Unit) -> Config -> PrestoDOM (Effect Unit) w
 startRide push config =
-  linearLayout
+  PrestoAnim.animationSet
+  [ scaleYAnimWithDuration 20
+  ]$ linearLayout
   [ width MATCH_PARENT
   , height (V 50)
   , background Color.darkMint
   , cornerRadius 8.0
-  , margin $ Margin 16 16 16 0
   , gravity CENTER
   , onClick push (const $ StartRide)
+  , pivotY 0.0
```

Comment:
if pivotY we are using for animation point, cant we animate without this ?

Reply:
`pivotY` defines where to start animation. I want to start from layout start so it required.